src/docfx/docfx.csproj

```diff
@@ -5,15 +5,20 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <LangVersion>Latest</LangVersion>
     <CodeAnalysisRuleSet>../../stylecop.ruleset</CodeAnalysisRuleSet>
+    <AssemblyName>docfx</AssemblyName>
+    <RootNamespace>Microsoft.Docs</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
   <ItemGroup>
     <AdditionalFiles Include="../../stylecop.json" />
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="1.0.210" />
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.0" PrivateAssets="All" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="Octokit" Version="0.29.0" />
```

Comment:
remove

Reply:
done

Reply:
1. Can we add comments in below of the author but not add a new comment?
2. Change the Comment status but not add a new `done` comment?

---
In reply to: [179977756](https://github.com/dotnet/docfx/pull/2601#discussion_r179977756) [](ancestors = 179977756)